### ISSUE TITLE
I have an issue with the name of your gem/class ;)

### ISSUE THREAD

**Issue by smathy** [mame/doublehelix#1]
Hi,

I created the "Acme" namespace here in the Ruby world for just such gems as yours.  Would you consider republishing it under the `Acme::DoubleHelix` name?

We already have: `Acme::Bleach`, `Acme::Heisenberg` and `Acme::Leeway` all in the tradition of Perl's Acme::\* libs.  All of which are listed as requirements of the base `Acme` gem for simple install.

If you're not interested, no problem, I'm happy to do the legwork for you, just thought I'd offer in case you were interested.

**Comment by mame** [comment]
I agree to clarify the category of gems.  However, I don't think that a namespace is a good way of that.  I guess few one will benefits even if I added just a line:

```
module Acme; module DoubleHelix; end; end 
```

to [this code](https://github.com/mame/doublehelix/blob/master/lib/doublehelix.rb).

Rather, RubyGems should have the feature.  It would be very good if the category information is included in gemspec and if we can filter gems with category specified:

```
gem search -c acme
```

**Comment by smathy** [comment]
So, you want to change RubyGems rather than namespace your gem?
